src/styles/arduino-dark.css

```diff
@@ -67,6 +68,7 @@
 .hljs-title,
 .hljs-section {
   color: #c678dd;
+  font-weight: normal;
```

Comment:
What is this trying to do? The default would already be normal...

Reply:
Yes, I know, I forget to delete...